index.html

```diff
@@ -32,96 +32,100 @@ <h3> I'm a software developer </h3>
       </div>     
   </section>
   <section class="works">
-    <div class="recent-works"> 
-      <h1> My recent works </h1>
+    <div class="recent-works">
+      <h1> My Recent Works</h1>
+      <div class="line"></div>
```

Comment:
- [ ] You did an amazing job but kindly following semantic HTML you are not allowed to use the h1 tag more than once. So please change it. 💪

Reply:
- [OPTIONAL] - I placed this as optional, but I **strongly** recommend following the advice of my previous reviewer and changing the tag from `h1` to `h2` or lower. 
I only make this requirement optional, so you won't need to repeat the week because of this.